src/qml/editorwidgets/Range.qml

```diff
@@ -50,6 +50,9 @@ Item {
     }
 
     onTextChanged: {
+      if ( value != undefined && value != text )
+        value = text
```

Comment:
Is this fixing a warning or something else?

Reply:
It is a warning I have been looking for quite some time already:

> Fix: FeatureForm.qml:348:11: QML Loader: Binding loop detected for property "value"